.github/workflows/dependabot-automerge.yml

```diff
@@ -0,0 +1,37 @@
+name: dependabot merger
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  review-pr:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' &&
+      github.event_name == 'pull_request' }}
+    permissions:
+      pull-requests: write
+      contents: write
+    steps:
+      - name: dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1.3.1
+        with:
+          github-token: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
`secrets.GITHUB_TOKEN` is automatically provided by the pipeline, right?

Reply:
This is also used in our Fosstars workflow, so presumably this works. If you have suggestions though on how to 100% confirm I'm happy to hear them :)